package.json

```diff
@@ -20,19 +20,20 @@
   "author": "Braintree <code@getbraintree.com> (https://www.braintreepayments.com/)",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.23",
-    "@typescript-eslint/eslint-plugin": "^5.19.0",
-    "eslint": "^8.13.0",
+    "@types/jest": "^29.2.3",
+    "@types/node": "^18.11.9",
```

Comment:
Let's only update this one if we're also going to bump the package's [.nvmrc](https://github.com/braintree/card-validator/blob/main/.nvmrc) to specify v18.
`@types/node` provides type definitions for NodeJS v18. Since we're on v16 (per `.nvmrc`) in this package, we'll want to keep those synced.